README.md

```diff
@@ -184,11 +184,12 @@ $ btcli unstake
 ```
 
 ### 4.2. Selecting the network to join 
-There are two open Bittensor networks: Nobunaga, Akatsuki, Nakamoto.
+There are two open Bittensor networks: Nobunaga, Akatsuki, Nakamoto, Local.
```

Comment:
While we're at it, can we take out "akatsuki" this is pretty old and we don't have it anymore?

Reply:
Do you want me to remove it from the selection prompt too, or just the README?